readthedocs/templates/projects/project_version_detail.html

```diff
@@ -15,7 +15,7 @@
 {% block content-header %}<h1>{% blocktrans with version.name as version_name %}{{ version_name }}{% endblocktrans %}</h1>{% endblock %}
 
 {% block content %}
-    <h2> Editing {{ version.slug }} </h2>
+    <h2> Editing {{ version.verbose_name }} </h2>
```

Comment:
I changed this so the user can see the _original_ name of the version. They already see the slug now.